pkg/services/sqlstore/migrations/accesscontrol/test/action_migrator_test.go

```diff
@@ -0,0 +1,241 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/services/dashboards"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	acmig "github.com/grafana/grafana/pkg/services/sqlstore/migrations/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/stretchr/testify/require"
+)
+
+func TestActionMigration(t *testing.T) {
+	// Run initial migration to have a working DB
+	x := setupTestDB(t)
```

Comment:
this test [runs only against `SQLite`](https://github.com/grafana/grafana/blob/f1b5e5b6b7e2f1e710038b39bc72f572fa77b183/pkg/services/sqlstore/migrations/accesscontrol/test/ac_test.go#L215); it would be better if it would check the `GRAFANA_TEST_DB` env variable similarly with other tests

Reply:
I've committed your patch to make these tests runnable against other DBs. Thanks for pointing it out!